outputs.tf

```diff
@@ -1,8 +1,12 @@
-#output "ecs_cluster" {
-#  value = ["${aws_subnet.private.*.id}"]
-#}
-
 output "cluster_id" {
   value = "${aws_ecs_cluster.cluster.id}"
 }
-# ECS servers
+
+
+output "autoscaling_group" {
+  value = {
+    id   = "${aws_autoscaling_group.ecs.id}"
+    name = "${aws_autoscaling_group.ecs.name}"
+    asrn = "${aws_autoscaling_group.ecs.arn}"
```

Comment:
:+1: on `arn`. When creating more complicated, grouped resources like this I prefer grouping related `outputs` as a `map`. It feels more resource-like when consumed as a module IMO. They're supported, so I'm not sure why it'd be bad if it's documented.